README.md

```diff
@@ -1,4 +1,6 @@
-# Gnosis Safe Android App [![Build Status](https://travis-ci.org/gnosis/safe-android.svg?branch=master)](https://travis-ci.org/gnosis/safe-android)
+# Gnosis Safe Android App 
+[![Build Status](https://travis-ci.org/gnosis/safe-android.svg?branch=master)](https://travis-ci.org/gnosis/safe-android)
+[![codecov](https://codecov.io/gh/gnosis/safe-android/branch/master/graph/badge.svg)](https://codecov.io/gh/gnosis/safe-android)
```

Comment:
14% 😢

Reply:
It seems not all tests are considered. E.g. codecov says that the settings view model is not tests, but there are tests for it https://github.com/gnosis/safe-android/blob/master/app/src/test/java/io/gnosis/safe/ui/safe/settings/SafeSettingsViewModelTest.kt

Could we investigate this.

Reply:
According to this it has some coverage: https://codecov.io/gh/gnosis/safe-android/pull/604/src/app/src/main/java/io/gnosis/safe/ui/safe/settings/SafeSettingsViewModel.kt

Reply:
That is the same coverage that I see in `app/build/reports/jacoco/createDebugTestCoverage/html/io.gnosis.safe.ui.safe.settings/SafeSettingsViewModel.kt.html` locally. After running: `./gradlew createDebugTestCoverage`

Reply:
let's see if that is fixed for master after the merge :)